web/src/app/workplace/tasklist/tasklist-toolbar/tasklist-toolbar.component.scss

```diff
@@ -3,11 +3,79 @@
   border: none;
 }
 
-.tab-align{
+.tab-align {
   padding: 8px 12px 8px 0px;
   margin-bottom: 0px;
 
-  &>div{
+  & > div {
     margin: 6px 0px;
   }
 }
+
+ul {
+  list-style-type: none;
+}
+
+.dropdown-menu > li > a, .dropdown {
```

Comment:
I think it can be refactored to a new file dropdown.scss and use this styling for all components which looks similar to this.